composer.json

```diff
@@ -13,7 +13,7 @@
         "illuminate/view": "5.*"
     },
     "require-dev": {
-        "mockery/mockery": "^0.9.5",
+        "mockery/mockery": "^1.3",
```

Comment:
Upgrading the `mockery/mockery` to fix following error during GitHub Actions building:

```
......
49) MailThiefTest::test_it_sends_mailables
Cannot use "parent" when current class scope has no parent

/Users/runner/work/mailthief/mailthief/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php:16
/Users/runner/work/mailthief/mailthief/vendor/mockery/mockery/library/Mockery/Loader/EvalLoader.php:16
/Users/runner/work/mailthief/mailthief/vendor/mockery/mockery/library/Mockery/Container.php:219
/Users/runner/work/mailthief/mailthief/vendor/mockery/mockery/library/Mockery.php:89
/Users/runner/work/mailthief/mailthief/tests/TestCase.php:11
/Users/runner/work/mailthief/mailthief/tests/TestCase.php:56
/Users/runner/work/mailthief/mailthief/tests/MailThiefTest.php:456
```